src/main/kotlin/org/rust/ide/colors/RsColors.kt

```diff
@@ -32,9 +32,11 @@ enum class RsColor(humanName: String, val default: TextAttributesKey) {
 
     CRATE("Crate", Default.IDENTIFIER),
     STRUCT("Struct", Default.CLASS_NAME),
+    STRUCT_COPYABLE("Copyable struct", Default.MARKUP_TAG),
```

Comment:
With new highlighting elements it would be a good idea to also update the color highlighting demo.
See RsColorSettingsPage.kt / highlighterDemoText.rs

Reply:
Thanks for reminding me